ArcGISRuntimeSDKQt_CppSamples/Layers/ListKmlContents/ListKmlContents.cpp

```diff
@@ -130,10 +129,10 @@ QString ListKmlContents::getKmlNodeType(KmlNode *node)
   default:
     return "";
   }
-  return " - " + type;
+  return type;
 }
 
-void ListKmlContents::displayChildren(KmlNode *parentNode)
```

Comment:
If we are just grabbing the children of the current node, I feel this does not need to be cached.

Again, this function is doing some surprising behaviour, we're wanting to list all the children for the given node, but this function also has logic to change `m_currentNode`. This does not feel like the remit of `displayChildren`?